### DISCUSSION TITLE
Signin with Apple Callback Error

### DISCUSSION THREAD

**Discussion by mstfash** [nextauthjs/next-auth#3484] [Help]
### Question 💬

I have 3 providers Facebook, Google and Apple but only the first 2 work correctly and apple does not, I authenticate fine, redirect url is okay and when I click on continue to sign me in, it redirects me to https://3bont.com/api/auth/signin?error=OAuthCallback this link with query string error OAuthCallback, Now I dunno what am I doing wrong, Could you help me out regarding that? Thanks in advance

  System:
    OS: macOS 10.15.7
    CPU: (12) x64 Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
    Memory: 575.03 MB / 16.00 GB
    Shell: 5.7.1 - /bin/zsh
  Binaries:
    Node: 16.13.0 - ~/.nvm/versions/node/v16.13.0/bin/node
    Yarn: 1.22.15 - ~/.yarn/bin/yarn
    npm: 8.1.4 - ~/.nvm/versions/node/v16.13.0/bin/npm
  Browsers:
    Chrome: 96.0.4664.110
    Firefox: 94.0.2
    Safari: 15.1
  npmPackages:
    next: latest => 10.0.3 
    next-auth: ^4.0.5 => 4.0.5 
    react: ^16.12.0 => 16.14.0 

### How to reproduce ☕️

- Go to my website https://3bont.com/account/login
- Signin with Apple
- It will not redirect you to https://3bont.com/api/auth/signin?error=OAuthCallback

### Contributing 🙌🏽

Yes, I am willing to help answer this question in a PR

**Comment by rahulmanas** [comment]
Getting the same error. Any help is really appreciated

**Reply by nikitalk** [reply]
I've faced with the same error. Apple auth is not working.

**Comment by emroot** [comment]
did you ever find a solution for this? Experiencing the same issue. Wish there was more support on this :(

**Comment by sanginkhidirov** [comment]
Hi, check this guid, its work for me:
https://www.kyle-melton.com/blog/2022-02-how-to-setup-sign-in-with-apple

**Comment by mehthabt** [comment]
any luck?

**Comment by MalvinJay** [comment]
Same issue happening to me. 
Though I have a staging site deployed on vercel which seems to work well there, on my main website, I keep getting `callbackUrl=[websiteRedirectPage]&error=OAuthCallback`.

Very Strange

**Reply by emibarea** [reply]
Hello! were you able to solve it?

**Reply by bibhuti9** [reply]
Any luck?
